### ISSUE TITLE
Adding a SERIAL column should be detected as blocking.

### ISSUE THREAD

**Issue by oomathias** [sbdchd/squawk#60]
```sql
ALTER TABLE a ADD COLUMN IF NOT EXISTS b SERIAL PRIMARY KEY;
```

This migration is blocking and is not detected as such. 

Is this something you could support?

**Comment by sbdchd** [comment]
Yup! shouldn't be too hard to add


AST that needs to be checked:

```json
❯ echo 'ALTER TABLE a ADD COLUMN IF NOT EXISTS b SERIAL PRIMARY KEY;' | squawk --dump-ast=parsed | jq .
[
  {
    "RawStmt": {
      "stmt": {
        "AlterTableStmt": {
          "cmds": [
            {
              "AlterTableCmd": {
                "subtype": 0,
                "name": null,
                "def": {
                  "ColumnDef": {
                    "colname": "b",
                    "typeName": {
                      "TypeName": {
                        "names": [
                          {
                            "String": {
                              "str": "serial"
                            }
                          }
                        ],
                        "typeOid": null,
                        "setof": false,
                        "pct_type": false,
                        "typmods": [],
                        "typemod": -1,
                        "arrayBounds": [],
                        "location": 41
                      }
                    },
                    "constraints": [
                      {
                        "Constraint": {
                          "contype": 5,
                          "location": 48,
                          "raw_expr": null,
                          "keys": null,
                          "indexname": null,
                          "skip_validation": false,
                          "initially_valid": false
                        }
                      }
                    ],
                    "is_local": true,
                    "location": 39
                  }
                },
                "behavior": 0,
                "missing_ok": true
              }
            }
          ],
          "relation": {
            "RangeVar": {
              "catalogname": null,
              "schemaname": null,
              "relname": "a",
              "inh": true,
              "relpersistence": "p",
              "alias": null,
              "location": 12
            }
          },
          "relkind": 37,
          "missing_ok": false
        }
      },
      "stmt_location": 0,
      "stmt_len": 59
    }
  }
]
```

**Comment by sbdchd** [comment]
Should be all set with the version 0.5.0

Let me know if you have any issues, suggestions etc

```
❯ echo "ALTER TABLE a ADD COLUMN IF NOT EXISTS b SERIAL PRIMARY KEY;" | squawk
stdin:1:0: warning: adding-serial-primary-key-field

   1 | ALTER TABLE a ADD COLUMN IF NOT EXISTS b SERIAL PRIMARY KEY;

  note: Adding a PRIMARY KEY constraint results in locks and table rewrites
  help: Add the PRIMARY KEY constraint USING an index.
```
